prov/efa/src/rxr/rxr_rma.c

```diff
@@ -43,19 +43,21 @@
 #include "rxr_read.h"
 
 int rxr_rma_verified_copy_iov(struct rxr_ep *ep, struct fi_rma_iov *rma,
-			      size_t count, uint32_t flags, struct iovec *iov)
+			      size_t count, uint32_t flags,
+			      struct iovec *iov, void **desc)
```

Comment:
Is `void *` the right type here? I see that the context we are inserting in `efa_mr_reg_impl` is type `fid_mr`.

Reply:
For libfabric API, a descriptor is of type ```void *```, and it is not necessary the memory registration pointer. It is that we (the EFA provider) choose to use MR pointer as descriptor. So the ```void*``` is right type here.

Reply:
I'm not sure this is true, from a quick scan I'm seeing `fid_mr`, `mem_desc`, and `efa_mr` used throughout the provider. I'm suspicious that we're doing the right thing due to the inconsistencies here.

Reply:
I am not sure I under stand the concern. The first member of ```struct efa_mr``` is ```struct fid_mr```. So ```efa_mr```, ```fid_mr``` and ```mem_desc``` are all pointing to the same memory address.

Reply:
I figured that was the case but this is really confusing.

Reply:
I think I understand you concern now. I should have called ```fi_mr_desc()``` to convert memory registration to descriptor in ```rxr_rma_verified_copy_iov```. but I did not. which caused confusion. I will update PR.

Reply:
Thanks, yep that's the concern. We can clean up the descriptor usage for other parts of the code in the future.

Reply:
PR updated.